.babelrc

```diff
@@ -3,7 +3,8 @@
     "env": {
       "development": {
         "plugins": [
-          "transform-es2015-modules-commonjs"
+          "transform-es2015-modules-commonjs",
+          "react-native-reanimated/plugin"
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding the "react-native-reanimated/plugin" to Babel plugins is necessary for using the react-native-reanimated library. However, please ensure that this plugin does not conflict with any other plugins or affect the performance of React Native animations in your application. If you encounter issues, consider isolating this plugin to only the parts of your codebase that require it.
```diff
  "plugins": [
    "transform-es2015-modules-commonjs",
+   "react-native-reanimated/plugin"
  ]
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->